lib/services/zendesk.rb

```diff
@@ -1,4 +1,6 @@
 class AhaServices::Zendesk < AhaService
   caption "Receive ideas from a Zendesk helpdesk"
   category "Customer relationship management"
+
+  ideas_portals
```

Comment:
Looking at how Salesforce _used_ to work, we have an `internal` attribute called `idea_portal_id` and that will end up rendering `app/views/settings/integrations/salesforce/_idea_portal_id.html.erb` in `aha-app`.

For this we should do the same, use the `internal` attribute here and simply copy the file in `aha-app` into a new Zendesk folder:

```
app/views/settings/integrations/zendesk/_idea_portal_id.html.erb
```

The file should be something like this:

```erb
<%= label_tag(:idea_portal_id, "Idea portal", class: "control-label") %>

<div class="controls">
  <%
    user_data = integration_user_data(integration)
    idea_portals = Account.current.idea_portals.permitted_through_portal_projects.select(:title, :id).order(:title).distinct
  %>

  <%=
    select_tag(
      "integration[attribute_idea_portal_id]", 
      options_for_select(
        idea_portals.map {|p| [p.title, p.id.to_s] },
        user_data.idea_portal_id.to_s
      ),
      include_blank: true
    )
  %>

  <span class="help-block">The Zendesk integration foo bar baz.</span>
</div>
```

And here we can simply use the internal attribute.

```suggestion
  internal :idea_portal_id
```

---

This will use the controller actions we already have and we won't need custom JS and a new IntegrationsController action.